packages/graphql-language-service-server/src/MessageProcessor.js

```diff
@@ -401,11 +401,20 @@ export class MessageProcessor {
     if (range) {
       position.line -= range.start.line;
     }
-    const result = await this._languageService.getAutocompleteSuggestions(
+    let result = await this._languageService.getAutocompleteSuggestions(
       query,
       position,
       textDocument.uri,
     );
+    if (textDocument.uri.endsWith('js')) {
```

Comment:
What about `jsx`, `ts` or `tsx`?

Reply:
Good point. I'm actually going to update this PR to handle this in a way that should be generic for all source file types (including graphql). Updates coming soon